tests/subsys/llext/src/test_llext_simple.c

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2023 Intel Corporation.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/ztest.h>
+#include <zephyr/kernel.h>
+#include <zephyr/llext/llext.h>
+#include <zephyr/llext/buf_loader.h>
+
+#ifdef CONFIG_ARM /* ARMV7 */
+const static uint8_t hello_world_elf[] = {
```

Comment:
while working on #62433 on Xtensa I hit a bug that cost me some time to track down - it turned out to be this misaligned test image. Not sure whether that can happen with ARM compilers too, but maybe add `__attribute__((aligned(4)));` here too